source/System.cpp

```diff
@@ -209,6 +211,12 @@ void System::Load(const DataNode &node, Set<Planet> &planets)
 			child.PrintTrace("Error: Expected key to have a value:");
 			continue;
 		}
+		else if(remove && child.Size() == 2 && key == "linger time")
+			lingerTime = -1;
+		else if(!remove && child.Size() == valueIndex + 1 && key == "linger time")
+			lingerTime = child.Value(valueIndex);
+		else if(key == "linger time")
+			child.PrintTrace("Invalid \"linger time\" specification.");
```

Comment:
prob should have both of these in the same else if(key == "linger time") for it to be a bit clearer

Reply:
That would add two lines, and another nesting level. People have preferred an else-if chain in the past.

Reply:
I could remove the "invalid linger time specification" check entirely, and it'll fall through to "skipping unrecognized attribute." Not as informative, but it does shrink the code.

Reply:
Your call, this isnt anything major

Reply:
It's gone now.